data/fwupd.service.in

```diff
@@ -12,8 +12,24 @@ RuntimeDirectory=@motd_dir@
 RuntimeDirectoryPreserve=yes
 BusName=org.freedesktop.fwupd
 ExecStart=@libexecdir@/fwupd/fwupd
-PrivateTmp=yes
+DevicePolicy=closed
+KeyringMode=private
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateDevices=no
+PrivateTmp=true
+ProtectClock=yes
+ProtectControlGroups=yes
 ProtectHome=yes
+ProtectHostname=yes
+ProtectKernelLogs=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
+ProtectProc=invisible
 ProtectSystem=full
-SystemCallFilter=~@mount
```

Comment:
Why is this dropped?

Reply:
This was inadvertently dropped while resorting the list.  Fixed